Lib/fontTools/merge.py

```diff
@@ -1018,13 +1122,16 @@ def merge(self, fontfiles):
 		allTags = reduce(set.union, (list(font.keys()) for font in fonts), set())
 		allTags.remove('GlyphOrder')
 
-		# Make sure we process cmap before GSUB as we have a dependency there.
+		# Make sure we process CFF before cmap, and cmap before GSUB as we have a dependency there.
```

Comment:
again unclear why cmap would depend on CFF

Reply:
No idea either, I removed the lines that adds `CFF` before all other tables and my test fonts merged just fine.

Reply:
@JeremieHornus do you remember why you did that?

Reply:
I honnestly don't remember… that was a while ago…

Reply:
no worries. Maybe it had something to do with glyph name collisions, when to-be-merged fonts contains same-named glyphs.. just a guess.

Reply:
yes, likely to be something like that…

Reply:
I’m testing with fonts that has duplicate glyph names and this change is not making any difference.